Pipfile

```diff
@@ -0,0 +1,50 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[requires]
+python_version = "3.5"
+
+[dev-packages]
+coveralls = ">=1.3.0"
+codecov = ">=2.0.15"
+pytest-cov = "==2.6.0"
+pytest-isort = "==0.2.1"
+pytest-mock = "==1.10.0"
+pytest-pylint = "==0.13.0"
+pytest-runner = "==4.2"
+pytest-dependency = "==0.4.0"
+yapf = "==0.24.0"
+docutils = "==0.14"
+crossbar = "==18.12.1"
+idna = "==2.5"
+onewire = "==0.2"
+pysnmp = "==4.4.6"
+pysnmp-mibs = "==0.1.6"
+xenavalkyrie = "==1.1.2"
+graphviz = "==0.8.4"
+Sphinx = "==1.8.1"
+sphinx_rtd_theme = "==0.4.1"
+pyModbusTCP = "==0.1.7"
+
+[packages]
+attrs = "==17.4.0"
+pexpect = "==4.6"
+pyserial = {git = "https://github.com/labgrid-project/pyserial.git", ref="v3.4.0.1", editable=true}
+pytest = "==4.0.2"
+pyudev = "==0.21.0"
+requests = "==2.19.1"
+xmodem = "==0.4.5"
+autobahn = "==18.12.1"
+ansicolors = "==1.1.8"
+"Jinja2" = "==2.10"
+PyYAML = "==3.13"
+"e1839a8" = {editable = true, path = "."}
```

Comment:
What is the meaning of this "magic" name?

Reply:
The magic name is automatically generated by pipenv for the labgrid project dependency. For better readability, i will change this to labgrid.